dgraph/cmd/bulk/run.go

```diff
@@ -155,10 +155,11 @@ func run() {
 		log.Fatal(http.ListenAndServe(opt.HttpAddr, nil))
 	}()
 
-	// Delete and recreate the output dirs to ensure they are empty.
-	x.Check(os.RemoveAll(opt.DgraphsDir))
+	// Delete and recreate the shard output dirs to ensure they are empty without destroying
+	// existing non-shard data, if any.
 	for i := 0; i < opt.ReduceShards; i++ {
 		dir := filepath.Join(opt.DgraphsDir, strconv.Itoa(i), "p")
+		x.Check(os.RemoveAll(dir))
```

Comment:
@codexnull This would remove the directory if it already exists. I don't think this is the behaviour we want.

Running 
```
./dgraph bulk -f ./goldendata.rdf.gz -s goldendata.schema --map_shards=4 --reduce_shards=2 --http localhost:8000 --zero=localhost:5080 --out=myDir
```
twice would overwrite the output created by the first bulk command.

The description of  https://github.com/dgraph-io/dgraph/issues/3026 says we don't want to remove the directory if it exists and has some data.